### PR TITLE
Added missing French translations to fr.js

### PR DESCRIPTION
**Pull request by belangervince**
1. What did you do? 

I added the missing translations in the fr.js file.

2. How did you do it?

Edited the files.

3. How should we verify this?

I have already submited missing translations in the past.

### CODE REVIEW

File: js/localization/messages/fr.json [hunk 1]
```diff
@@ -289,86 +289,158 @@
...
+        "dxDiagram-commandFillColor": "Couleur remplissage",
+        "dxDiagram-commandAlignLeft": "Aligner à gauche",
+        "dxDiagram-commandAlignCenter": "Centrer",
+        "dxDiagram-commandAlignRight": "Aligner à droite",
+        "dxDiagram-commandConnectorLineType": "Type de ligne de connexion",
+        "dxDiagram-commandConnectorLineStraight": "Droit",
+        "dxDiagram-commandConnectorLineOrthogonal": "Orthogonal",
+        "dxDiagram-commandConnectorLineStart": "Début de la ligne de connexion",
+        "dxDiagram-commandConnectorLineEnd": "Fin de la ligne de connexion",
+        "dxDiagram-commandConnectorLineNone": "Ligne",
```
Comment (BingoRUS) [inline js/localization/messages/fr.json]: @belangervince Should it be `Aucun` here?

Comment (belangervince) [reply js/localization/messages/fr.json]: Yes, you're correct. I just commited the change

Comment (BingoRUS) [reply js/localization/messages/fr.json]: Thank you! Once the unit tests pass, I will merge this PR. The updated French messages will be available with the next DevExtreme release (v19.2.4).

Comment (belangervince) [reply js/localization/messages/fr.json]: Thank you!

**Review by BingoRUS — APPROVED** [review]

**Pull request merged**